docs/spec/tendermint-fork-cases/TendermintAcc3.tla

```diff
@@ -438,10 +426,49 @@ AgreementOrEquivocationOrAmnesiaOld ==
 AgreementOrEquivocationOrAmnesia ==
     \/ Agreement
     \/ \E Detectable \in SUBSET Faulty:
-        /\ Cardinality(Detectable) >= THRESHOLD1
+        /\ Cardinality(Detectable) = THRESHOLD1
```

Comment:
This is super important, as we need to argue in the English spec that we can punish `T+1` bad validators. Perhaps we should rename the property to something with `accountability`. This property needs to be references in the English spec

Reply:
Jawohl!